### PR TITLE
Filter refresh and postal code editing

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
HCAP-527 and 520 are back with a vengeance. 

- MoH can now update participants postal codes. This also updates their FSA
- Fixed issue where the filter would reset after being idle for 5 minutes
- I had to lint ignore a warning to get this to work. I really think we should rework the SiteTable to use less useEffect hooks

### CODE REVIEW

File: client/src/pages/private/SiteTable.js [hunk 1]
```diff
@@ -136,25 +135,15 @@ export default ({ sites }) => {
     };
 
     fetchUserInfo();
+
+    // This fetch sites is a dependency of this function. This needs to be reworked, but it is outside of the scope of the ticket
+    /* eslint-disable */
```
Comment (arranfw) [inline client/src/pages/private/SiteTable.js:140]: Can we do a disable next line here with the specific rule?

File: client/src/pages/private/SiteTable.js [hunk 2]
```diff
@@ -231,7 +220,9 @@ export default ({ sites }) => {
           <Grid item>
             <Box minWidth={180}>
               <TableFilter
-                onFilter={(filteredRows) => setRows(filteredRows)}
+                onFilter={(filteredRows) => {
```
Comment (arranfw) [inline client/src/pages/private/SiteTable.js:223]: were these brackets added intentionally?

Comment (dbayly-freshworks) [reply client/src/pages/private/SiteTable.js:223]: I can remove them. As part of my debugging I had a console.log here.

**Review by arranfw — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]
👍

**Review by ashtonmeuser — APPROVED** [review]

**Comment by ashtonmeuser** [comment]
We intend to do a release containing only email blast changes to prod. This is on hold until we've done so. See [discussion](https://freshworks.slack.com/archives/C011G18PW5S/p1622576835021600).

File: client/src/pages/private/ParticipantTable.js [hunk 3]
```diff
@@ -600,6 +600,9 @@ export default () => {
             onSubmit={async (values) => {
               if (values.phoneNumber && Number.isInteger(values.phoneNumber))
                 values.phoneNumber = values.phoneNumber.toString();
+              if (values.postalCode && values.postalCode.length > 3) {
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantTable.js:603]: should this be >= in the case of them inputting only the three characters of the postal code?

File: client/src/pages/private/SiteTable.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ const columns = [
   { id: 'details' },
 ];
 
-export default ({ sites }) => {
+export default () => {
```
Comment (Zenen-freshworks) [inline client/src/pages/private/SiteTable.js:25]: nice, this is cleaner

**Pull request merged**